domains/tanish.chahal.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Tanish's Portfolio",
+    "repo": "https://github.com/TanishChahal/register",
+    "owner": {
+        "username": "TanishChahal",
+        "email": "TanishChahal5705@gmail.com"
+    },
+    "record": {
+        "URL": "tanishchahal.netlify.app"
```

For URL records you need to add the protocol
```suggestion
        "URL": "https://tanishchahal.netlify.app"
```